### ISSUE TITLE
Suggestion: Pickaxe side vector?

### ISSUE THREAD

**Issue by wompking** [SamsTheNerd/HexGloop#60]
When mining using a Hex Pickaxe, the position of the block mined is pushed to the stack before the contained hexes are executed. It might be helpful if the side normal vector is also pushed to the stack, so casters don't have to mess with potentially inaccurate workarounds like Architect's Distillation.

**Comment by NovaStorm93** [comment]
yeah this would be useful with the pickaxe (also help reduce media usage), the location where the item was dropped is not helpful with a lot of spells, the direction it was broken from would be nice
